sdk/search/search-documents/src/serviceUtils.ts

```diff
@@ -365,6 +370,54 @@ export function generatedIndexToPublicIndex(generatedIndex: GeneratedSearchIndex
   };
 }
 
+export function generatedSearchResultToPublicSearchResult<T>(results: GeneratedSearchResult[]) {
+  const returnValues: SearchResult<T>[] = [];
+  results.forEach((result) => {
```

Comment:
Array.map is a little easier than forEach and pushing to a new Array yourself

```suggestion
  const returnValues = results.map<SearchResult<T>>((result) => {
     // create obj in here and return it
  });
```

Reply:
Done